button style 2.css

```diff
@@ -0,0 +1,28 @@
+.btn {
+	background: none;
+	border: none;
+	padding: 10px 20px;
+	font-size: 20px;
+	margin: 10px;
+	transition: 0.8s;
+	position: relative;
+	cursor: pointer;
+}
+.btn:hover {
+	color: white;
+}
+.btn::before {
```

Comment:
Mandou bem no `::before`. Use sempre `::before` e `::after` quando for criar conteúdo meramente decorativo, que não tem relevância para o conteúdo do site, já que google e outros robos ignoram tudo que está dentro dessas tags.